bindings/go/src/fdb/fdb.go

```diff
@@ -145,9 +145,9 @@ func APIVersion(version int) error {
 		return errAPIVersionNotSupported
 	}
 
-	if e := C.fdb_select_api_version_impl(C.int(version), C.int(headerVersion)); e != 0 {
-		if e != 0 {
-			if e == 2203 {
+	if err := C.fdb_select_api_version_impl(C.int(version), C.int(headerVersion)); err != 0 {
```

Comment:
This one should be `errorCode` or similar, IMO, because it's not a Go error but rather a C function return value.